src/OMSimulatorLib/System.cpp

```diff
@@ -1689,6 +1692,9 @@ bool oms::System::copyResources()
 
 void oms::System::getAllResources(std::vector<std::string>& resources) const
 {
+  if (!values.empty() && !Flags::ExportParametersInline())
+    resources.push_back("resources/" + std::string(this->getFullCref()) + ".ssv");
```

Comment:
the ssv file can be pushed directly in `exportoFile()` in `Model.cpp` which is easily understandable i guess

Reply:
values contain all the values that will be exported to the ssv file, right? Therefore, it would make sense to move everything ssv related to that class. I am not quite sure why it didn't work, but I would still like to move those things there.